molecule/cookiecutter/scenario/driver/ec2/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/create.yml

```diff
@@ -59,10 +59,19 @@
         mode: 0600
       when: keypair.changed
 
+    - name: Get the ec2 ami(s) by owner and name, if image not set
```

Comment:
Can you also please add this to `test/resources/playbooks/ec2/create.yml` for our functional testing?

Reply:
In which AWS region do the tests run? The AWS_REGION environment variable is set in Travis CI, right?

Reply:
They don't run at all :sweat_smile: 

We'll figure it out when we get there, just set whatever for now :)

Reply:
I've just pushed the required change for functional testing.

Wouldn't be better defining the region directly in the create playbook?